ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/virtual-source-nat.j2

```diff
@@ -6,4 +6,4 @@
 ip address virtual source-nat vrf {{ vrf }} address {{ virtual_source_nat_vrfs[vrf].ip_address }}
 {%         endif %}
 {%     endfor %}
-{% endif %}
\ No newline at end of file
+{% endif %}
```

Comment:
Empty new line is missing

Reply:
there is one, you just can see it on Github review!

Reply:
There is one you just can't see in GitHub review! I've been caught by this before as well... when there is no new line you see a (-) red sign on the last line. It was commit c9aee95